### DISCUSSION TITLE
[Question] Does not work .exe file. what is problem? pleasse help me

### DISCUSSION THREAD

**Discussion by jjjunehkim** [flet-dev/flet#3401] [Q&A] [answered]
### Question

when I just run by "python main.py" , it work.

I created .exe file by using "flet build windows"
"requirement.txt"file  is in   flet==0.22.*

flet verison => 0.22.1

flutter version ==> 
 Flutter 3.22.1 • channel stable • https://github.com/flutter/flutter.git
Framework • revision a14f74ff3a (9 days ago) • 2024-05-22 11:08:21 -0500
Engine • revision 55eae6864b
Tools • Dart 3.4.1 • DevTools 2.34.3

OS ==> windows 11 pro

But  when I run .exe file,  it doesn't work,
when I ran exe file, it was just opened and displayed noting and then after 3~5 second, it was closed. 

what is problem?  please help me.

P.S.  I tried antother.
1. I made new flet scipt automatically by "flet create test"

   below code is created automatically by command("flet create test")
 
    import flet as ft
    def main(page: ft.Page):
          page.add(ft.SafeArea(ft.Text("Hello, Flet!")))
    ft.app(main)

3.  and create .exe file by "flet build window"
4. run .exe file -> result is same(just opened and displayed noting and after a few second, closed.  

### Code sample

```python
import flet as ft


def main(page : ft.Page):

    page.theme_mode = ft.ThemeMode.DARK 

    page.vertical_alignment = ft.MainAxisAlignment.CENTER   
    page.horizontal_alignment = ft.CrossAxisAlignment.CENTER
    

    page.window_resizable = False  
    page.expand = True
    page.padding = 0

    page.add(
        ft.Container(
            content = ft.Column(
                controls=[
                    ft.Text("TEST 1"),
                    ft.Text("TEST 2"),
                    ft.Text("TEST 3"),
                ]
            )
        )
    )


ft.app(target = main, assets_dir="assets")
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by jjjunehkim** [comment, marked as answer]
I got  a solution from  moderator of our language channel in Discord 
Problem was flutter version....

**Reply by OtherwisePitt** [reply]
paste here please

**Reply by jjjunehkim** [reply]
I did downgrade flutter version from 3.22.1 to 3.22.0.
